.gitignore

```diff
@@ -1,6 +1,8 @@
 #==============================================================================#
 # Files to ignore.
 #==============================================================================#
+/tests/benchmarks*
```

Comment:
just curios: are the files created by testing framework? If so, we also need to fix that.

Reply:
> just curios: are the files created by testing framework? If so, we also need to fix that.

Nope, it just personal usage to test some models :)

Reply:
Could you remove it? we might put a "benchmarks" subdirectory under tests :)

Otherwise, this patch looks good.